spyder/plugins/updatemanager/widgets/update.py

```diff
@@ -48,6 +49,14 @@
 HEADER = _("<h3>Spyder {} is available!</h3><br>")
 URL_I = 'https://docs.spyder-ide.org/current/installation.html'
 
+SKIP_CHECK_UPDATE = (
+    sys.executable.startswith(('/usr/bin/', '/usr/local/bin/'))
+    or (
+        not is_anaconda()
```

Comment:
Shouldn't we use here `is_conda_env`? You can import it from Spyder-kernels.

Reply:
That's a good question.

We use `is_anaconda` throughout Spyder's source code and it does essentially the same thing, so it does duplicate code, which is not ideal. I would suggest that we use `is_conda_env` from `spyder-kernels` exclusively and remove `is_anaconda` from the Spyder source code. Is there a compelling reason to keep `is_anaconda`?

Reply:
To be clear, I recommend removing `is_anaconda` in a separate PR

Reply:
> I would suggest that we use is_conda_env from spyder-kernels exclusively and remove is_anaconda from the Spyder source code. Is there a compelling reason to keep is_anaconda?

`is_anaconda` came first and it's a simpler check because it doesn't depend on an interpreter. But I agree, we could remove it and leave `is_conda_env` instead, to make things simpler.

> To be clear, I recommend removing is_anaconda in a separate PR

Ok, I think that's a good idea.

----

On a second thought, I think this is the right check in this case because we need to run it only for Spyder's interpreter. So, sorry for the noise.